Automattic/wp-calypso
# Focused Launch: add a resuable summary item

#### Changes proposed in this Pull Request

This PR adds a reusable item for the summary page in focused launch. It also temporarily adds all possible usage combinations for the item for demoing purposes. These will be deleted once this PR is reviewed. 

#### Testing instructions

1. In apps/editing-toolkit run `yarn dev --sync`.
2. Sandbox testingfocusedflow.wordpress.com.
3. Visit https://testingfocusedflow.wordpress.com/wp-admin/post-new.php.
4. In console, type `wp.data.dispatch('automattic/launch' ).openFocusedLaunch()`.
5. You should see all combination rendered. 

Fixes #47147
